lib/heimdall_tools/nessus_mapper.rb

```diff
@@ -221,8 +221,12 @@ def to_hdf
           end
           if item['compliance-reference']
             @item['tags']['nist']     = cci_nist_tag(parse_refs(item['compliance-reference'], 'CCI'))
+            @item['tags']['cci']      = parse_refs(item['compliance-reference'], 'CCI')
+            @item['tags']['rid']      = parse_refs(item['compliance-reference'], 'Rule-ID').join(',')
+            @item['tags']['Stig_id']  = parse_refs(item['compliance-reference'], 'STIG-ID').join(',')
```

Comment:
are we sure `'Stig_id'` is the correct way this should be represented? In inspec_tools I've been using `'stigid'`